backoffice/src/Helpers/SearchHelpers/DroitQueryBuilder.cs

```diff
@@ -94,12 +94,15 @@ public static IQueryable<Droit> BuildQuery(DroitSearchForm form, IQueryable<Droi
                 (form.IsIsolatedFind == null || d.WreckId == null == form.IsIsolatedFind) &&
                 (form.AssignedToUserId == null || d.AssignedToUserId == form.AssignedToUserId)
             );
+            query = query.Where(d => 
+                form.StatusList.IsNullOrEmpty() ||
+                form.StatusList.Contains(d.Status));
+            
             query = query.Where(d =>
-                ( form.StatusList.IsNullOrEmpty() ||
-                  form.StatusList.Contains(d.Status) ));
-            query = query.Where(d =>
-                ( form.TriageNumbers.IsNullOrEmpty() ||
-                  (d.TriageNumber!= null && form.TriageNumbers.Contains(d.TriageNumber.Value) )));
+                form.TriageNumbers.IsNullOrEmpty() ||
```

Comment:
Zero is basically a non-triage number they can select for ones without a Triage number set